tests/pillar/system.sls

```diff
@@ -75,11 +107,11 @@ linux:
       opencontrail:
         source: "deb http://ppa.launchpad.net/tcpcloud/contrail-2.20/ubuntu trusty main"
         architectures: amd64
-    policyrcd:
-      - package: cassandra
-        action: exit 101
-      - package: '*'
-        action: switch
+    #policyrcd:
```

Comment:
kitchen will not pass anyway - tested locally, not part of this commit., it this case salt complains for missing priority to be set. Policyrcd had a different issues I challenged. I want to have a look on this on the kitchen commit.